### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by GraceMattern**
- `/` supports many devices, responsive
- `/forum` supports many devices, responsive
- `/application/:id` delete button should look normal
- `/application/:id` icons for mobile are bigger
- added salary field on add job app forms (affects dashboard and sidebar)
- add `/offers` to sidebar
- update sidebar add job application to be a modal not /create
- remove `/offers/:cycleId` from client readme
- remove `/application/:id/:cycleId` from client readme
- add `/create` to client readme

### CODE REVIEW

File: client/src/api/routes/applications.ts [hunk 1]
```diff
@@ -43,14 +43,15 @@ export const useGetApplicationFromCycle = (cycleId: string, applicationId: strin
  * @returns {UseQueryResult<ApplicationObject>} the newly created application
  * @throws if fails
  */
-export const useCreateApplication = (company: string, position: string, location: string, jobPostUrl: string, description: string, applyDate: Date): UseQueryResult<ApplicationObject> => {
+export const useCreateApplication = (company: string, position: string, location: string, jobPostUrl: string, description: string, salary: number, applyDate: Date): UseQueryResult<ApplicationObject> => {
```
Comment (mpolimen) [inline client/src/api/routes/applications.ts]: salary should be removed from the useCreateApplication stuff, since the backend doesn't use it

Comment (mpolimen) [reply client/src/api/routes/applications.ts]: The forms should also not have it, they won't know salary until they get an offer anyway, most of the time.

**Review by mpolimen — COMMENTED** [review]
Saw these warnings when running:

WARNING in src/components/SideDrawer/index.tsx
  Line 60:12:  'built' is assigned a value but never used           @typescript-eslint/no-unused-vars
  Line 71:18:  'cycleData' is assigned a value but never used       @typescript-eslint/no-unused-vars
  Line 71:40:  'CycleIsLoading' is assigned a value but never used  @typescript-eslint/no-unused-vars
  Line 71:65:  'CycleIsError' is assigned a value but never used    @typescript-eslint/no-unused-vars

src/pages/Job/index.tsx
  Line 74:9:  'iconSize' is assigned a value but never used  @typescript-eslint/no-unused-vars

**Comment by mpolimen** [comment]
Otherwise, the responsiveness looks much nicer site wide, except for the metrics page (totally my fault since I made the page)

**Comment by GraceMattern** [comment]
- `/application/:id` add margins to contacts form
- `/dashboard` make content a link not the box containing the content and bold title
- `/application/:id` uphold figma design for position and company font colors
- remove unused imports
- `/signup` add caption to password field
- adjust fontsize for comically large error message

File: client/src/pages/Job/index.tsx [hunk 2]
```diff
@@ -361,21 +360,21 @@ export default function Job() {
...
                         <Typography variant="h2" id="company"
                             sx={{ fontSize: h2Size }}
-                            color="#ADA7BD">{data.company}</Typography>
+                            color="#190446">{data.company}</Typography>
                     </div>
                     {/* delete button */}
-                    <Button variant="contained"
+                    <Button 
+                        variant="contained"
                         onClick={async () => {
```
Comment (bwormser) [inline client/src/pages/Job/index.tsx:371]: The delete button seems awkwardly big and the icon isn't centered

**Review by bwormser — COMMENTED** [review]
Overall looks good except for my one comment

**Pull request merged**